### PR TITLE
Translated guides/concepts/static-site-generation.md to Spanish

### PR DESCRIPTION
**Pull request by FelixMor**
Translated guides/concepts/static-site-generation.md to Spanish

### CODE REVIEW

**Comment by debs-obrien** [comment]
Hi @FelixMor great work. Can you check your pr and remove all things that shouldn't be there please. You will also have to resolve conflicts. When that is done perhaps @alvarosaburido or @Dawntraoz can give it a quick review. Thanks

**Comment by debs-obrien** [comment]
Also @FelixMor I just noticed that there is another pr with this file translated. Perhaps you can review this pr and ask for changes in that pr and close this one perhaps. The title of pr says it is for only one file yet 7 files. Always good to keep things as simple as possible. But seen as you have worked on this page it would be great if you could review this https://github.com/nuxt/nuxtjs.org/pull/763 Thanks for the great work

File: content/es/guides/concepts/static-site-generation.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    correctAnswer: Falso
+  - question: ¿Qué comando se debe utilizar para generar un sitio web estático?
+    answers:
+      - nuxt build
+      - nuxt prerender
+      - nuxt generate
+    correctAnswer: nuxt generate
+  - question: ¿En qué momentos se llama a la API?
+    answers:
+      - Cada vez que navegas hacia la página del el contenido de la API
```
Comment (alvarosaburido) [inline content/es/guides/concepts/static-site-generation.md:20]: `del` en lugar de `del el`

File: content/es/guides/concepts/static-site-generation.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      - Cuando generas el sitio
+      - Cuando generas el sitio y cada vez que navegues a la página del contenido de la API 
+    correctAnswer: Cuando generas el sitio
+  - question: ¿Qué páginas vuelven al modo de aplicación de página única?
```
Comment (alvarosaburido) [inline content/es/guides/concepts/static-site-generation.md:24]: Al ser un término técnico (SPA): 

- ¿Qué páginas vuelven al modo de SPA? 
- ¿Qué páginas vuelven al modo de aplicación de página única (SPA, _single page aplication_)? 

 @Dawntraoz que opinas?

File: content/es/guides/features/nuxt-components.md [hunk 3]
```diff
@@ -0,0 +1,425 @@
...
+
+<base-alert type="info">
+
+`<NuxtLink>` posee [*smart prefetching*](/guides/features/nuxt-components#the-nuxtlink-component) integrado.
+
+</base-alert>
+
+## prefetchLinks
+
+Nuxt.js incluye *smart prefetching* de forma automática. Esto significa que detecta cuando un hipervínculo es visible, ya sea en el viewport o al desplazarse y realiza el *prefetching* en el JavaScript para todas aquellas páginas, para que estén listas cuando el usuario da clic al hipervínculo. Nuxt.js solamente carga los recursos cuando el navegador no se encuentra ocupado y no realiza el *prefetching* si tu conexión está offline(fuera de línea) o si solamente tienes conexión 2g.
```
Comment (alvarosaburido) [inline content/es/guides/features/nuxt-components.md:264]: Nuxt.js incluye *smart prefetching* (precarga inteligente)

File: pages/guides/_book/_slug.vue [hunk 4]
```diff
@@ -147,6 +147,7 @@ export default {
     }
   }
 }
+// test comment bla bla bla
```
Comment (alvarosaburido) [inline pages/guides/_book/_slug.vue:150]: Please remove

**Review by alvarosaburido — CHANGES_REQUESTED** [review]
@FelixMor great work, just small changes.

**Comment by alvarosaburido** [comment]
> Hi @FelixMor great work. Can you check your pr and remove all things that shouldn't be there, please. You will also have to resolve conflicts. When that is done perhaps @alvarosaburido or @Dawntraoz can give it a quick review. Thanks

Hey @debs-obrien, sorry 😅,  I was on holidays haha, by the way, there are some binary files like `static/blog/build-dev-to-clone-with-nuxt-new-fetch/.dev-clone-nuxt.mp4.icloud `  that shouldn't be there, those are the ones you're referring too?

File: content/es/guides/concepts/static-site-generation.md (hunk 1 above)
Comment (Kronhyx) [reply content/es/guides/concepts/static-site-generation.md:20]:
```suggestion
      - Cada vez que navegas hacia la página del contenido de la API
```

File: content/es/guides/concepts/static-site-generation.md (hunk 2 above)
Comment (Kronhyx) [reply content/es/guides/concepts/static-site-generation.md:24]: @alvarosaburido Apoyo más la segunda variante.

```suggestion
  - question: ¿Qué páginas vuelven al modo de aplicación de página única (SPA, single page aplication)?
```

File: content/es/guides/features/nuxt-components.md (hunk 3 above)
Comment (Kronhyx) [reply content/es/guides/features/nuxt-components.md:264]:
```suggestion
Nuxt.js incluye *smart prefetching* (precarga inteligente) de forma automática. Esto significa que detecta cuando un hipervínculo es visible, ya sea en el viewport o al desplazarse y realiza el *prefetching* en el JavaScript para todas aquellas páginas, para que estén listas cuando el usuario da clic al hipervínculo. Nuxt.js solamente carga los recursos cuando el navegador no se encuentra ocupado y no realiza el *prefetching* si tu conexión está offline(fuera de línea) o si solamente tienes conexión 2g.
```

File: pages/guides/_book/_slug.vue (hunk 4 above)
Comment (Kronhyx) [reply pages/guides/_book/_slug.vue:150]:
```suggestion
```

File: content/es/guides/features/nuxt-components.md [hunk 5]
```diff
@@ -261,7 +261,7 @@ Para aprender más sobre `<RouterLink>`, consulta la [Documentación sobre Vue
 
 ## prefetchLinks
 
-Nuxt.js incluye *smart prefetching* de forma automática. Esto significa que detecta cuando un hipervínculo es visible, ya sea en el viewport o al desplazarse y realiza el *prefetching* en el JavaScript para todas aquellas páginas, para que estén listas cuando el usuario da clic al hipervínculo. Nuxt.js solamente carga los recursos cuando el navegador no se encuentra ocupado y no realiza el *prefetching* si tu conexión está offline(fuera de línea) o si solamente tienes conexión 2g.
+Nuxt.js incluye *smart prefetching* (precarga inteligente) de forma automática. Esto significa que detecta cuando un hipervínculo es visible, ya sea en el viewport o al desplazarse y realiza el *prefetching* en el JavaScript para todas aquellas páginas, para que estén listas cuando el usuario da clic al hipervínculo. Nuxt.js solamente carga los recursos cuando el navegador no se encuentra ocupado y no realiza el *prefetching* si tu conexión está offline(fuera de línea) o si solamente tienes conexión 2g.
```
Comment (nilsandrey) [inline content/es/guides/features/nuxt-components.md:264]: I propose some minor adjustments:

> Nuxt.js incluye precarga inteligente (*smart prefetching*) de forma automática. Esto significa que detecta cuando un hipervínculo es visible, ya sea en el viewport o al desplazarse y realiza la precarga en JavaScript de esa página, para que esté lista cuando el usuario de clic en el hipervínculo. Nuxt.js solamente carga los recursos cuando el navegador no se encuentra ocupado y no realiza la precarga si estás desconectado (offline) o si solamente tienes conexión 2g.

**Comment by debs-obrien** [comment]
closing this as other pr was merged. please make any further changes in another pr. thanks

**Pull request closed without merging**